sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -188,15 +188,15 @@ def _send(self, message, timeout=None, last_exception=None):
         except Exception as e:
             raise MessageSendFailed(e)
 
-    def _schedule(self, message, schedule_time_utc):
-        # type: (Union[Message, BatchMessage], datetime.datetime) -> List[int]
-        """Send Message or BatchMessage to be enqueued at a specific time.
+    def schedule(self, messages, schedule_time_utc):
+        # type: (Union[Message, List[Message]], datetime.datetime) -> List[int]
+        """Send Message or multiple Messages to be enqueued at a specific time.
         Returns a list of the sequence numbers of the enqueued messages.
-        :param message: The messages to schedule.
-        :type message: ~azure.servicebus.Message or ~azure.servicebus.BatchMessage
+        :param messages: The message or list of messages to schedule.
+        :type messages: ~azure.servicebus.Message or List[~azure.servicebus.Message]
         :param schedule_time_utc: The utc date and time to enqueue the messages.
         :type schedule_time_utc: ~datetime.datetime
-        :rtype: List[int]
+        :rtype: list[int]
```

Comment:
Nit: capitalization was changed?

Reply:
So for type hint: `List` is defined in `typing` module, which supports Generics -- `List[SOME_TYPE]` while `list` not supporting Generics.

But in the docstring, `list` is the real type we're accepting.
That's the difference I found after I bing it for a while.

Reply:
I think List instead of list for typing? The standard typing library uses List